src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -9,18 +9,18 @@
 using System.Windows;
 
 [assembly: AssemblyTitle("DynamoCoreWpf")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Autodesk, Inc.")]
-[assembly: AssemblyProduct("DynamoCoreWpf")]
-[assembly: AssemblyCopyright("Copyright © Autodesk, Inc. 2015")]
-[assembly: AssemblyTrademark("")]
+//[assembly: AssemblyDescription("")]
+//[assembly: AssemblyConfiguration("")]
+//[assembly: AssemblyCompany("Autodesk, Inc.")]
+//[assembly: AssemblyProduct("DynamoCoreWpf")]
+//[assembly: AssemblyCopyright("Copyright © Autodesk, Inc. 2015")]
+//[assembly: AssemblyTrademark("")]
```

Comment:
Please remove them completely instead of commenting them out, it creates questions.